source/jormungandr/jormungandr/instance.py

```diff
@@ -222,6 +222,11 @@ def factor_too_long_journey(self):
         return get_value_or_default('factor_too_long_journey', instance_db, self.name)
 
     @property
+    def bus_physical_mode_id(self):
```

Comment:
Since it is parametrized, I think I would have parametrized a `successive_physical_mode_limit` to be able to limit some the number of trains or planes taken